CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ Releasing is documented in RELEASE.md
 ### Removed
 
 ### Fixed
+- avoid routing on roads with access reserved to customers ([#2060](https://github.com/GIScience/openrouteservice/pull/2060))
```

Comment:
Usually, we use the issue-id instead of the PR-id in the changelog.

Reply:
Actually, I've seen both. While usually it doesn't make much of a difference, it still can happen that:
- although a PR fixes or implements the functionality described in an issue it actually goes beyond that description such that it is necessary to read through the PR's description in order to comprehend it fully 
- there is no corresponding issue to a PR
- a single PR tackles multiple issues

Therefore, I personally prefer to include the PR-id. Please note that PR ids is also what ends up in the auto-generated release notes.

Reply:
ok.